pkg/cli/admin/release/info.go

```diff
@@ -1987,6 +2035,306 @@ func describeBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string, format s
 	return nil
 }
 
+func (o *InfoOptions) listRpmdb(releaseInfo *ReleaseInfo, dir, format, targetImage string) error {
+	if targetImage == "" {
+		// find image for machine-os component
+		if tag, ok := releaseInfo.ComponentTags["machine-os"]; !ok {
+			return fmt.Errorf("component machine-os not found in release")
+		} else {
+			targetImage = tag
+		}
+	}
+
+	var err error
+	var targetTag *imageapi.TagReference
+	for _, tag := range releaseInfo.References.Spec.Tags {
+		if tag.Name == targetImage {
+			targetTag = &tag
+			break
+		}
+	}
+	if targetTag == nil {
+		return fmt.Errorf("tag %s not found in release", targetImage)
+	}
+
+	ref, err := imagereference.Parse(targetTag.From.Name)
+	if err != nil {
+		return err
+	}
+
+	// not sure this can ever happen, but just sanity-check anyway
+	if ref.ID == "" {
+		return fmt.Errorf("image not digest-based: %s", ref)
+	}
+
+	dbpath := filepath.Join(dir, ref.ID)
```

Comment:
nit: `ID` is a digest, and [all digests will contain `:`][1].  Non-POSIX systems (OS X?  Windows?) might have trouble with that, and things like [the OCI image-layout format][1] suggest splitting the digest, using the algorithm as a directory, and the encoded portion as the child filename.  But will anyone be running this with caching outside of POSIX?  Maybe we don't care enough to be worth coding up that kind of hoop-jumping?

[1]: https://github.com/opencontainers/image-spec/blob/v1.0.0/descriptor.md#digests
[2]: https://github.com/opencontainers/image-spec/blob/v1.0.0/image-layout.md#blobs

Reply:
Yeah, IIRC I basically just did this to match `describeBugs()` here. No strong opinion one way or the other though I guess we should try to be consistent unless there's a good reason to differ.

Reply:
Good idea, I think that makes sense to do regardless.

Reply:
Given that (as you found out), this currently requires `rpm` anyway, I think for now let's confine these commands to Linux. As mentioned in the commit message, there's potential with SBOM that this becomes much more lightweight and as a result also cross-platform.

Reply:
Tweaked this now to just substitute `:` for `_` to be compliant.